other/get.webgl.org/troubleshooting/DoNotCopyOrLinkThisFileElseYouWillNotGetAutoUpdatedHelpForYourUsers.js

```diff
@@ -148,35 +148,35 @@ var BrowserDetect = {
     subString: "Win",
     identity: "Windows",
     browsers: [
-      {url: "http://www.mozilla.com/en-US/firefox/new/", name: "Mozilla Firefox"},
-      {url: "http://www.opera.com/", name: "Opera"},
-      {url: "http://www.google.com/chrome/", name: "Google Chrome"},
-      {url: "http://www.microsoft.com/en-us/windows/windows-10-upgrade ", name: "Edge"},
+      {url: "https://www.mozilla.org/en-US/firefox/new/", name: "Mozilla Firefox"},
+      {url: "https://www.opera.com", name: "Opera"},
+      {url: "https://www.google.com/chrome/", name: "Google Chrome"},
+      {url: "https://www.microsoft.com/en-us/windows/get-windows-10", name: "Edge"},
       {url: "http://www.microsoft.com/ie", name: "Internet Explorer"}
     ]
   },
   { string: navigator.platform,
     subString: "Mac",
     identity: "Mac",
     browsers: [
-      {url: "http://www.mozilla.com/en-US/firefox/new/", name: "Mozilla Firefox"},
-      {url: "http://www.google.com/chrome/", name: "Google Chrome"},
-      {url: "http://www.opera.com/", name: "Opera"},
-      {url: "http://www.webkit.org/", name: "WebKit Developer Builds"}
+      {url: "https://www.mozilla.org/en-US/firefox/new/", name: "Mozilla Firefox"},
+      {url: "https://www.google.com/chrome/", name: "Google Chrome"},
+      {url: "https://www.opera.com", name: "Opera"},
+      {url: "https://webkit.org", name: "WebKit Developer Builds"}
```

Comment:
I'm not sure this makes sense any more. Due to code signing changes on macOS these developer builds basically don't work any more. It may be worth changing this to Safari Technology Preview at https://developer.apple.com/safari/technology-preview/ .